.buildkite/scripts/setup_old_deps.sh

```diff
@@ -9,5 +9,3 @@ apt-get update
 apt-get install -y python3.5 python3.5-dev python3-pip libxml2-dev libxslt-dev xmlsec1 zlib1g-dev tox
 
 export LANG="C.UTF-8"
```

Comment:
this export goes nowhere. I suspect it was here for a reason; you'll need to figure out if that reason was important.